src/test-util.ts

```diff
@@ -56,6 +59,18 @@ export class TestToolCache {
   }
 }
 
+export const TEST_TMP_FILES_DIR = path.join(os.tmpdir(), 'test-setupcloudsdk');
+
+/**
+ * Helper to write test data to disk.
+ */
+export async function writeTmpFile(data: string): Promise<string> {
```

Comment:
We probably want to use `writeSecureFile` instead: https://github.com/google-github-actions/auth/blob/9a051ab76ce8a17efc6be51a4f66a0248970f317/src/utils.ts#L13

Reply:
That will also drop the dep on uuid

Reply:
Good idea, reused that and removed uuid.